pkg/cosign/verify.go

```diff
@@ -697,33 +680,83 @@ func verifyInternal(ctx context.Context, sig oci.Signature, h v1.Hash,
 		return bundleVerified, nil
```

Comment:
So the plan for the next PR is to move the above TSABundle verification below, and use the time from it?

Does providing a signed timestamp supersede the timestamp in the Rekor bundle? Or do we check both and warn the certificate can be verified with one and not the other?

Reply:
Actually, I hadn't considered that, but I like that: let's verify with both, and expect both to pass. 

Eventually we can phase out Rekor time or something, but for now we expect both to work, so let's validate against both.

Reply:
I'll follow-up in the next PR